### PR TITLE
Sep7 UI Tweaks

### PR DESCRIPTION
**Pull request by ddworken**
This addresses the majority of PICNIC-145:

[x] Rounded modal corners on desktop
[x] Delete back button on desktop
[x] BodyTiny → BodySmall 
[x] Disable pay button if they haven't entered an amount yet
[ ] Scrolling
[x] Left align recipient
[x] Margins around stellar address
[x] Black stellar logo
[x] Blue highlight for pending transaction

Note the change for the last one that makes it so all pending transactions are highlighted in blue.

### CODE REVIEW

File: shared/wallets/transaction/index.tsx [hunk 1]
```diff
@@ -74,7 +74,11 @@ type CounterpartyTextProps = {
 export const CounterpartyText = (props: CounterpartyTextProps) => {
   switch (props.counterpartyType) {
     case 'airdrop':
-      return <Text style={{color: globalColors.white}} type={props.textTypeSemibold}>Stellar airdrop</Text>
+      return (
```
Comment (nathunsmitty) [inline shared/wallets/transaction/index.tsx]: I ran into a similar autoformatting thing in #18214, think your merge conflicts are probably from that

**Comment by nathunsmitty** [comment]
Note that we'll need to cherry-pick this in, see #releases

File: shared/wallets/transaction/index.tsx (hunk 1 above)
Comment (ddworken) [reply shared/wallets/transaction/index.tsx]: :+1: Fixed the merge conflict

**Comment by ddworken** [comment]
Rounded corners: 

image

Blue highlight: 

image

Margins and disabled pay button:

image

Enabled pay button:

image

**Comment by cjb** [comment]
@ddworken Thanks! Could we keep the memo/message/etc vertically aligned instead of indented?  Here's the Zeplin shot:

Screen Shot 2019-07-02 at 8 11 39 AM

**Pull request merged**